src/class-elements/grammar-privatename-no-receiver-error-accessor-get-meth.case

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: It is syntax error if IdentifierName is accessed without receiver (Accessor get Method)
+info: |
+  Updated Productions
+
+  ClassElementName :
+    PropertyName
+    PrivateName
+
+  PrivateName ::
+    # IdentifierName
+template: syntax/invalid
+features: [class-methods-private]
+---*/
+
+//- elements
+get #m() {}
+method() {
+#m();
```

Comment:
I don't have time for a full review today but one thing that might be interesting here is to limit the reference only to the get operation. `#m` should be enough.